app/controllers/dashboard/districts_controller.rb

```diff
@@ -27,6 +28,10 @@ def show
 
   private
 
+  def feature_flag_required
+    redirect_to root_url unless current_admin.feature_enabled?(:dashboard_v2)
```

Comment:
404 if we wanna be super watertight, not give away that a feature is hidden here? :smile: